rules/cloud/azure/azure_ad_user_added_to_admin_role.yml

```diff
@@ -0,0 +1,23 @@
+title: User added to an administrator's Azure AD role
```

Comment:
`title: User Added to an Administrator's Azure AD Role`
cf : https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#title